### PR TITLE
Distinguish keys in validator lists.

### LINKED ISSUE #1364: Duplicate keys on mystake page
The keys need to be prefixed as a validator may be in both lists:

image

### PR DESCRIPTION
**Pull request by NodeGuy**
Closes #1364 

Description:

Use unique v-for keys in validator lists in My Delegations.  I couldn't reproduce the bug (#1364) but this should prevent it from recurring.

### CODE REVIEW

File: app/src/renderer/components/staking/TabMyDelegations.vue [hunk 1]
```diff
@@ -221,13 +228,13 @@ h3
 
 .check-out-message
   background var(--app-fg)
+  border 1px solid var(--bc-dim)
```
Comment (faboweb) [inline app/src/renderer/components/staking/TabMyDelegations.vue]: using some style formatting plugin?

Comment (jbibla) [reply app/src/renderer/components/staking/TabMyDelegations.vue]: yes. he auto-sorts it alphabetically. it's beautiful. 😢 <-- tears of joy

Comment (faboweb) [reply app/src/renderer/components/staking/TabMyDelegations.vue]: can prettier do this?

Comment (NodeGuy) [reply app/src/renderer/components/staking/TabMyDelegations.vue]: No, [Prettier doesn't support Stylus](https://github.com/prettier/prettier/issues/2465).

Comment (jbibla) [reply app/src/renderer/components/staking/TabMyDelegations.vue]: what i'm hearing is that we all want to switch to plain css 😉 

no his editor has a plugin or something like that.

Comment (NodeGuy) [reply app/src/renderer/components/staking/TabMyDelegations.vue]: It's [Stylus Supremacy](https://thisismanta.github.io/stylus-supremacy/#).  Good news @jbibla, there's an Atom plugin!

Comment (jbibla) [reply app/src/renderer/components/staking/TabMyDelegations.vue]: the best way to format stylus is to write css :trollface:

**Pull request merged**